### PR TITLE
Update unicode_smilies.php

### PR DESCRIPTION
**Pull request by loma-one**
:p in re:pubica was interpreted as emoji and output in text

### CODE REVIEW

File: unicode_smilies/unicode_smilies.php [hunk 1]
```diff
@@ -14,586 +14,583 @@ function unicode_smilies_install() {
...
-	Smilies::add($b, ':-O', '&#x1F62E;');
-	Smilies::add($b, ':O', '&#x1F62E;');
-	Smilies::add($b, ':-X', '&#x1F910;');
-	Smilies::add($b, ':-x', '&#x1F910;');
-	Smilies::add($b, ':X', '&#x1F910;');
-	Smilies::add($b, ':x', '&#x1F910;');
-	Smilies::add($b, '8-)', '&#x1F60E;');
-	Smilies::add($b, ':-/', '&#x1F615;');
-	Smilies::add($b, ':-"', '&#x1F48F;');
+	Smilies::add($b, ':-):', '&#x1F600;');
```
Comment (tobiasd) [inline unicode_smilies/unicode_smilies.php]: While it is correct that the emoticon names are enclosed by a pair of `:` at the beginning and the end, I don't think the same should be done with plain ASCII emoticons.

I think the problem with the `:p` you mentioned is rather, that two letter representations of the emoticons should not be replaced at all as their combination is to common in the normally used language.

Comment (loma-one) [reply unicode_smilies/unicode_smilies.php]: changed: The plain ASCII emoticons are not enclosed by a :

File: unicode_smilies/unicode_smilies.php [hunk 2]
```diff
@@ -45,555 +45,552 @@ function unicode_smilies_smilies(&$a,&$b) {
...
+//	Smilies::add($b, ':\\.../:', '&#x;');
+//	Smilies::add($b, ':\\ooo/:', '&#x;');
+//	Smilies::add($b, ':headdesk:', '&#x;');
 
 // These two are still in core, so oldcore isn't strictly right, but we don't want too many directories
 
-//	Smilies::add($b, ':-d', '&#x;');
-	Smilies::add($b, ':-o', '&#x1F62E;');
+//	Smilies::add($b, ':-d:', '&#x;');
+	Smilies::add($b, ':-o:', '&#x1F62E;');
```
Comment (MrPetovan) [inline unicode_smilies/unicode_smilies.php]: This doesn't look right either, both of these are meant to represent the actual smiley. Please either remove the trailing colon or disable the replacement altogether.

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: unicode_smilies/unicode_smilies.php (hunk 2 above)
Comment (loma-one) [reply unicode_smilies/unicode_smilies.php]: changed

**Review by MrPetovan — APPROVED** [review]

**Comment by MrPetovan** [comment]
Thank you for your work!

**Pull request merged**